docs/Spec.md

```diff
@@ -0,0 +1,74 @@
+### Spec
+
+---
+
+### Feature Definitions
+
+[Filter of GitHub Features](https://github.com/fsheremetyeva/capstone/labels/Feature)
+
+- Ability to create user volunteer account
+- Ability to create non-profit organization account
+- Ability to search for volunteer opportunities
+- Initial Dashboard for volunteer account
+- Initial Dashboard for non-profit account
```

Comment:
What are the capabilities of this dashboard? We can see the following being feature possibilities:

- [ ] Add/Update/Remove Volunteer Opportunities
- [ ] Upcoming Events
- [ ] Recently Completed Events
- [ ] New Volunteers

Reply:
At this point only the first item is a part of my plan. But if the time permits, I will explore the other options. Thanks.

Reply:
Added to Project Spec